### this message is to tell DM if the map is built
# time 
time  stamp    
# built or not
bool map_built